### ISSUE TITLE
Catch `operator.div` with `--py3k`

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#1936]
### Steps to reproduce
1. Save the following to a file and lint it
```
import operator
print(operator.div)
```

### Current behavior
We don't emit anything.

### Expected behavior
We should warn that `operator.div` is gone in Python 3.

### pylint --version output
master
